src/test/java/uk/gov/companieshouse/api/strikeoffobjections/email/KafkaRestClientUnitTest.java

```diff
@@ -1,4 +1,4 @@
-package email;
+package uk.gov.companieshouse.api.strikeoffobjections.email;
```

Comment:
You should now be able to remove the import statement below (for KafkaRestClient). This should be flagged by Sonar I'd have thought.